instrumentation/httpclient-4.0/src/main/java/org/apache/http/client/HttpClient.java

```diff
@@ -63,6 +63,7 @@ private static void processResponse(URI requestURI, HttpResponse response) {
                 .uri(requestURI)
                 .procedure(PROCEDURE)
                 .inboundHeaders(inboundCatWrapper)
+                .status(response.getStatusLine().getStatusCode(), response.getStatusLine().getReasonPhrase())
```

Comment:
This is what every instrumentation will have to add.
The step builder still works because status() was added to the Build interface which is the last step.

A previous design used a new interface that extended Build. But it would break if a client assigned the steps to variables. e.g.
```
UriParameter uriParam = HttpParameters.library("gRPC");
ProcedureParameter procParam = uriParam.uri(uri);
// supposing the new interface was returned by uri(), the line above would break.
```